dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V8.0_2024.03.07__set_eperson_process_nullable.sql

```diff
@@ -0,0 +1,9 @@
+--
```

Comment:
@Bezkup : In order to backport this to 7.6.x (as a bug fix), we would need to **rename** the two SQL migration scripts in this PR to start with `V7.6_*`... For example:
* `V7.6_2024.03.07__set_eperson_process_nullable.sql`

This would need to be done for BOTH the postgres and h2 script.   The reason for this change is that 8.0 will run *any older script* while 7.6.x will want to have a script that starts with `V7.6`

Reply:
Okay, done!